__tests__/app.test.js

```diff
@@ -134,3 +134,75 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("Status: 200 responds with an array of article objects containing author, title, article_id, topic, created_at, votes and comment_count", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toEqual(expect.any(Array));
+        expect(Object.keys(body.articles[0])).toEqual(
+          expect.arrayContaining([
+            "article_id",
+            "title",
+            "topic",
+            "author",
+            "created_at",
+            "votes",
+            "comment_count",
+          ])
+        );
+      });
+  });
+});
```

Comment:
It looks like you have branched this from your ticket 8 branch before you made some corrections - this is why we always branch from main!
Be wary of marge conflicts when merging this branch

Reply:
oops sorry..